### PR TITLE
[ML] Rename increased_memory_estimate_bytes

### PR DESCRIPTION
**Pull request by dimitris-athanasiou**
... to memory_reestimate_bytes in DF Analytics
memory usage.

Relates #58588

### CODE REVIEW

File: docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc [hunk 1]
```diff
@@ -444,6 +444,11 @@ The memory usage status. May have one of the following values:
 * `hard_limit`: usage surpassed the configured memory limit.
 --
 
+`memory_reestimate_bytes`::::
```
Comment (droberts195) [inline docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc:447]: I think the fields in each section are supposed to be in alphabetical order.  That would put this one first within the collapsible section.

**Review by droberts195 — APPROVED** [review]
LGTM

File: docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc (hunk 1 above)
Comment (dimitris-athanasiou) [reply docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc]: Fixed

**Comment by dimitris-athanasiou** [comment]
run elasticsearch-ci/2

**Pull request merged**